src/reducers/nations.js

```diff
@@ -72,4 +72,12 @@ export default function (state = initialState, action) {
 }
 
 export const openedNation = state => resolveNation(state.nations, state.openedNationId);
-export const isDraft = nation => resolveStatus(nation) === 'draft';
+export const isDraft = (nation) => {
```

Comment:
Just check if the smart contract's id is `<= -1` and if the `nation.tx === null`. If the tx is null, it mean's that it was not submitted to the blockchain.

Reply:
I'd like to have all status determination logic in one place, so I move it to utils/nations.js.